src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -30,6 +34,10 @@ public class Book {
     private String isbn;
     @Column(nullable = false)
     private BigDecimal price;
+    @Column
```

Comment:
You don't use this, I think you can remove it

Reply:
Agree